tests/static/js/hours/open-status/messagefactory.js

```diff
@@ -0,0 +1,50 @@
+import { OpenStatusStrings, OpenStatusTypes } from '../../../../../static/js/hours/open-status/constants';
+import OpenStatusMessageFactory from '../../../../../static/js/hours/open-status/messagefactory';
+
+const localizer = {
+  getLocalizedTime: jest.fn(yextTime => {
+    let time = new Date();
+    time.setHours(Math.floor(yextTime / 100));
+    time.setMinutes(yextTime % 100);
+
+    return time.toLocaleString('en', {
+      hour: 'numeric',
+      minute: 'numeric'
+    });
+  }),
+  getTranslation: jest.fn(str => {
+    const translations = {
+      [OpenStatusStrings.CLOSED]: 'Closed',
```

Comment:
Can we use strings different than the values of the `OpenStatusStrings` object? Right now I think this technically does not test the regression being fixed

Reply:
So because it verifies the expected calls to `getTranslation` are being made, it does cover this specific regression. We did not wrap one of the strings in a `getTranslation` call. The assertions on line 47 and 48 make sure we have two calls to that method, meaning all strings are wrapped.

Reply:
Oh I see I missed that, I feel like it would be a little simpler to just change the mocked value of getTranslation to be an actualy translation instead of adding extra expects but that also works!